aws-serverless-js-httpendpoint/index.js

```diff
@@ -0,0 +1,62 @@
+const pulumi = require("@pulumi/pulumi")
+const aws = require("@pulumi/aws")
+const serverless = require("@pulumi/aws-serverless");
+
+// A table to hold the counts of each route that is hit. We use the route name
+// as the key for the the table.  The read and write capacity are intentionally
+// small for this toy example.  The AWS Free Teir provides 25 units of read and
+// write capacity across all DynamoDB Tables in your account.
+const table = new aws.dynamodb.Table("counterTable", {
+    attributes: [{
+        name: "route",
+        type: "S",
+    }],
+    hashKey: "route",
+    readCapacity: 1,
+    writeCapacity: 1,
+});
+
+// The handler for GET /{route+}.
+const getHandler = async (event) => {
```

Comment:
In our other JS examples, we recently decided to avoid `async/await` and use promises directly instead, so that examples will work on earlier Node versions.  That might be a little more painful here, but probably worth being consistent.